.gitignore

```
@@ -0,0 +1,16 @@
+# Serverless directories
+.serverless
+package-lock.json
+
+# golang output binary directory
+bin
```

Does this need a trailing slash?
```suggestion
bin/
```

---

I don't think it does, since it works without it.  That means, though, that I shouldn't have added a .gitkeep file to it.

---

good, maybe a slash would just restrict it to _only_ directories